plugins/dependencies/src/main/kotlin/de/fayard/refreshVersions/internal/KotlinPoetry.kt

```diff
@@ -0,0 +1,103 @@
+package de.fayard.refreshVersions.internal
+
+import com.squareup.kotlinpoet.*
+
+
+internal data class Dependency(
+    val group: String = "",
+    val module: String = "",
+    val version: String = ""
+) {
+    val name: String get() = module
+    fun groupModuleVersion() = "$group:$module:$version"
+    fun groupModuleUnderscore() = "$group:$module:_"
+    fun groupModule() = "$group:$module"
+    fun versionName(mode: VersionMode): String = PluginConfig.escapeLibsKt(
+        when (mode) {
+            VersionMode.MODULE -> module
+            VersionMode.GROUP -> group
+            VersionMode.GROUP_MODULE -> "${group}_$module"
+        }
+    )
+
+    override fun toString() = groupModuleVersion()
+}
+
+
+internal class Deps(
+    val dependencies: List<Dependency>,
+    val modes: Map<Dependency, VersionMode>,
```

Comment:
This is not used according to the IDE. Why is it there, then?

Reply:
deleted